Meta/jbig2_to_pdf.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+"""
+Creates a PDF that embeds a jbig2 image. Useful for viewing .jbig2 files in
+PDF viewers, since all PDF viewers support .jbig2 but few image viewers do.
+
+Usage is a bit clunky (use Build/lagom/bin/file to get the dimensions):
+% Meta/jbig2_to_pdf.py -o foo.pdf path/to/bitmap.jbig2 399 400
+% open foo.pdf
+"""
+
+import argparse
+import sys
+import textwrap
+
+
+def dedent(b):
+    return textwrap.dedent(b.decode('latin1')).encode('latin1')
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        epilog=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    parser.add_argument("image", help="Input image")
+    parser.add_argument("width", type=int, help="image width")
+    parser.add_argument("height", type=int, help="image height")
+    parser.add_argument("-o", "--output", help="Path to output PDF")
+    args = parser.parse_args()
+
+    width, height = args.width, args.height
+
+    with open(args.image, 'rb') as f:
+        image_data = f.read()
+        print('dims %dx%d' % (width, height))
```

Comment:
Can we use f-strings instead of the old string formatting?

Reply:
Sure, will do, one sec…

Reply:
Actually, there are no f-strings for bytestrings, apparently.

Reply:
Here's my local diff that I made before I realized this: [fbstrings.patch](https://github.com/SerenityOS/serenity/files/14624721/fbstrings.patch) https://stackoverflow.com/questions/45360480/is-there-a-formatted-byte-string-literal-in-python-3-6 talks about this – kinda disappointing that this doesn't exist!

Reply:
…I did change this `print()` though, since that doesn't use `bytes`.

Reply:
Oh okay, that's weird. You should be able to use the solution proposed on stackoverflow, right? Use strings and encode to bytes at the very end. (I'm okay with both approaches)

Reply:
It's possible, but PDFs morally are bytes, not strings. (I admittedly do transcode to be able to use `textwrap.dedent()`, but that's hidden as an implementation detail.) So that seems like a hack around a language deficiency to me. It seems better to me to use a language feature that actually supports bytestrings.